recipes/libcoro/all/conanfile.py

```diff
@@ -40,8 +40,12 @@ def _min_cppstd(self):
     @property
     def _minimum_compilers_version(self):
         return {
-            "gcc": "10.2.1",
-        }
+                "gcc": "10.2.1",
+                "clang": "16.0.0",
+                "apple-clang": "13",
```

Comment:
```suggestion
                "apple-clang": "16",
```

Need to re-check it with the upstream, but probably the project needs an update.